test/OpenTelemetry.Exporter.Geneva.Tests/LogSerializationTests.cs

```diff
@@ -45,16 +45,57 @@ public LogSerializationTests(ITestOutputHelper output)
     public void SerializationTestForException()
     {
         var exceptionMessage = "Exception Message";
-        var exportedFields = GetExportedFieldsAfterLogging(logger => logger.Log<object>(
-            logLevel: LogLevel.Information,
-            eventId: default,
-            state: null,
-            exception: new Exception(exceptionMessage),
-            formatter: null));
-
-        PrintFields(this.output, exportedFields);
+        var exStack = "Exception StackTrace";
+        var ex = new MyException(exceptionMessage, exStack);
+        var exportedFields = GetExportedFieldsAfterLogging(
+            logger =>
+            {
+                logger.Log<object>(
+                            logLevel: LogLevel.Information,
+                            eventId: default,
+                            state: null,
+                            exception: ex,
+                            formatter: null);
+            },
+            (genevaOptions) =>
+            genevaOptions.ExceptionStackExportOption = ExceptionStackExportOptions.ExportAsString);
```

Comment:
You could probably update this test to a `Theory` to test both the exception stack export options.